translation/en/RedirectedResources/assets/abdata/h/list/00_00/personality_voice_c00_03_00/translation.txt

```diff
@@ -613,11 +613,11 @@
 //おちんちんもゴムもまだ抜かないで……余韻を味わいましょ=
 //思いっきり楽しめた？　私はもう大満足よ……=
 //あっ……ゴムの先がまた私の奥をツンツンしてる=
-//私にこんなに出してくれてる……嬉しいわ=
+//私にこんなに出してくれてる……嬉しいわ=You poured out a lot inside me... I'm glad~
 //君の気持ちはちゃんと受け取ったわ……ココにね♪=
 //ザーメンの熱さが心地いいわ……=
 //あん、こぼれちゃわないように蓋をしてて……君のおチンポで♪=
-//君の想いがあふれてきちゃう……もったいないわ=
+//君の想いがあふれてきちゃう……もったいないわ=Your love is overflowing a lot... What a waste...
```

Comment:
Okay, I would leave it with "love" but in case there are similar lines that use the same expression, just to be sure, I'll change it to "feelings" as you suggest.